requirements.sh

```diff
@@ -248,18 +248,20 @@ elif apt --version 2>/dev/null; then
 	apt install -y cmake g++ make build-essential autoconf automake uuid-dev
 	apt install -y libtool libboost-dev libboost-system-dev libboost-thread-dev libpq-dev libz-dev
 	apt install -y python-dev python3-dev python3-pip python3-numpy
+        pip3 install --upgrade pip
 
 	sqlite3_build_prepare
 	make
-
+        
+	apt install -y sqlite3 # make install after sqlite3_build_prepare should be enough to install sqlite3 as a command 
 	apt install -y pkg-config
 
 	# for Kerberos authentication, avoid interactive questions
 	DEBIAN_FRONTEND=noninteractive apt install -yq krb5-user
 	DEBIAN_FRONTEND=noninteractive apt install -yq libcurl4-openssl-dev
 
-	apt install -y cpulimit
+	apt install -y cpulimit 
+	
 else
-	echo "Requirements cannot be automatically installed, please refer README.rst to install requirements manually"
-yum install -y rh-python36
```

Comment:
python36 package does not require on yum?

Reply:
This is in else block which is meant for unsupported env. Don't why it was added in no yum /RedHat family block